app/assets/javascripts/actions/notifications.js

```diff
@@ -0,0 +1,77 @@
+import actionTypes from './actionTypes';
+import status from 'http-status';
+
+export const addNotification = (notification) => ({
+  type: actionTypes.ADD_NOTIFICATION,
+  notification
+});
+
+export const removeNotification = (id) => ({
+  type: actionTypes.REMOVE_NOTIFICATION,
+  id
+});
+
+export const sendSuccessNotification = (message) =>
+  (dispatch, getState, { Notification }) => {
+    const newNotification = Notification.createNotification({
+      type: Notification.types.SUCCESS,
+      message
+    });
+
+    dispatch(addNotification(newNotification));
+
+    setTimeout(() => {
+      dispatch(removeNotification(newNotification.id));
+    }, 4000);
+  }
+
+export const sendErrorNotification = (error) =>
+  (dispatch, getState, { Notification }) => {
+    const type = Notification.types.ERROR;
+
+    switch (error.response.status) {
```

Comment:
Here we're assuming that the error was caused by a completed HTTP request and that the server was able to be reached, we should avoid it because if the connection of the user fails it's not gonna work. When the server can't be reached there will be no `error.response` so it's gonna try to access `status` attribute of `undefined`, can we cover this case?